docs/debug-launch.md

```diff
@@ -0,0 +1,97 @@
+# CMake: Debug and launch
+
+CMake Tools makes it easier to set up debugging. Because C and C++ projects may define multiple (sometimes dozens or even hundreds) of executables, creating a `launch.json` may be difficult.
+
+If you define any executable targets via CMake, CMake Tools will be aware of them and allow you to start debugging them.
+
+> **Note:**
+> Debugging is only supported when CMake is in  _CMake Server_ mode. This mode is enabled automatically for CMake versions 3.7.2 and above. It is not available on older versions.
```

Comment:
This should be "...when CMake is using CMake server mode or file-based API. These modes are enabled automatically...." (or something to show that debugging is supported by both CMake server mode and file-based API)

Reply:
I've changed this to read: Debugging is supported when CMake is using either _CMake Server_ or the cmake-file-api. These modes are enabled automatically for CMake versions 3.7.2 and above. Debugging is not available on older versions.